test/extended/networking/network_segmentation.go

```diff
@@ -4,6 +4,9 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/pointer"
```

Comment:
nit: I think it should be move to k8s dependencies group. (starting at line 22)

Reply:
fixed